.gitignore

```diff
@@ -30,5 +30,4 @@ dist
 
 *.ngfactory.ts
 *.ngsummary.json
-yarn.lock
-package-lock.json
+yarn.lock
```

Comment:
Is committing the package-lock.json recommended practice now?

Reply:
Yep!

Reply:
👍